models/model.js

```diff
@@ -54,7 +54,26 @@ exports.selectCommentsByArticleId = (id) => {
         );
       }
     })
-    .then(({rows}) => {
+    .then(({ rows }) => {
       return rows;
     });
 };
+
+exports.insertCommentByArticleId = (id, usernameAndBody) => {
+  const insertQuery =
+    "INSERT INTO comments (article_id, author, body) VALUES ($1, $2, $3) RETURNING *;";
+  const { username, body } = usernameAndBody;
+  const params = [id, username, body];
+
+  return db.query(insertQuery, params).then(({ rows }) => {
+    if (typeof body !== "string") {
```

Comment:
I feel like this if statement could be done before the query is done to save us from doing an unnecessary query